src/layouts/Nav/nav.module.styl

```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
   padding-left $navbar-padding-vertical
   padding-right $navbar-padding-vertical
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
+
+
+.hamburger
+  flex 1
+  align-self center
+  @media (min-width: 760px)
+    visibility hidden
```

Comment:
using display none removes the element from the DOM altogether, which means there are only 2 flex elements in the Nav. The result is the title is no longer centered.